python/py-gst-python/Portfile

```diff
@@ -21,19 +21,20 @@ checksums           rmd160  38becce2998c4d5909bc3493c124d850925de473 \
                     sha256  853ea35a1088c762fb703e5aea9c30031a19222b59786b6599956e154620fa2f \
                     size    153428
 
-python.versions     36 37 38 39 310
-
-depends_build       port:pkgconfig
-depends_lib         port:py${python.version}-gobject3 \
-                    port:gstreamer1-gst-plugins-base
+python.versions     38 39 310
 
 use_configure       yes
+configure.pkg_config_path ${python.prefix}/lib/pkgconfig
 
-configure.pkg_config_path  ${python.prefix}/lib/pkgconfig
-configure.pre_args         --prefix='${python.prefix}'
-configure.args             -Dpython=${python.bin}
-configure.cppflags-append  -I${python.prefix}/include
+configure.pre_args-append --prefix='${python.prefix}'
+configure.args-append     -Dpython=${python.bin}
+configure.cppflags-append -I${python.prefix}/include
 
 destroot.cmd        ${build.cmd}
 
+if {${name} ne ${subport}} {
+depends_build       port:pkgconfig
+depends_lib         port:py${python.version}-gobject3 \
```

Comment:
please always use `depends_build-append` and `depends_lib-append` so that you are not overwriting dependencies that are added by the PortGroupts

Reply:
also, please indent these lines properly